### PR TITLE
Fix/datetimejsonencoding

### PR DESCRIPTION
**Pull request by surajrav**
Since we discovered that `datetime` objects were being stored as strings in MongoDB, and subsequently fixed them to be stored instead as MongoDB's native DateTime objects, the service gets back a `datetime.datetime` type for all of the datestamp fields.

Such a type is not natively JSON serializable and hence this PR adds a custom JSON encoder to convert that into a serializable string when going over the NATS bus.

While here I also took the liberty to modify some absolute imports to relative ones (only the ones that reference local miteD sources within themselves. This facilitates easier development when using the direct sources with a middleware/service for X1.

### CODE REVIEW

**Review by pmercado23 — APPROVED** [review]
LGTM 🚢

File: setup.py [hunk 1]
```diff
@@ -12,9 +12,9 @@
     author='Harry Winters',
     author_email='harry.winters@twoporeguys.com',
     install_requires=[
-        'requests==2.19.1',
-        'sanic==0.7.0',
+        'requests==2.20.1',
```
Comment (HarryMWinters) [inline setup.py:15]: Why all the bumps here? Seems like it risks breakage unnecessarily. Also, do we still need `schematics` with the validators gone?

File: validators/validators.py [hunk 2]
```diff
@@ -1,129 +0,0 @@
-import logging
```
Comment (HarryMWinters) [inline validators/validators.py:1]: Good nuke.

**Review by HarryMWinters — COMMENTED** [review]
Some comments. Nothing major but I'd be curious for an answer.

File: setup.py (hunk 1 above)
Comment (surajrav) [reply setup.py:15]: oops, schematics should go right now.

I mean this is still not a `prod` release and we do have tests, so was wondering that I should take the chance and update them now before we enter more serious prod cycles.

I'll nuke the schematics req... good catch

**Comment by surajrav** [comment]
@HarryMWinters new version `2.1.1` made, tagged, and pushed to nexus ... please check again and merge if you deem fit

**Review by HarryMWinters — APPROVED** [review]
LGTM. Merging.

**Pull request merged**